gems/aws-sdk-core/spec/aws-sdk-core/shared_config_file_factory_spec.rb

```diff
@@ -3,7 +3,7 @@
 require_relative '../spec_helper'
 
 module AWS::SDK::Core
-  describe SharedConfigFileFactory do
+  describe ConfigFileFactory do
```

Comment:
With this name change, probably need to update the spec file names as well.

Reply:
I think I did but didn't commit yet